app/editor/src/features/admin/sources/SourceForm.tsx

```diff
@@ -31,27 +32,24 @@ const SourceForm: React.FC<ISourceProps> = (props) => {
   const { isShowing, toggle } = useModal();
 
   const sourceId = Number(id);
-  const [source, setSource] = React.useState<ISourceModel>((state as any)?.source ?? defaultSource);
+  const [source, setSource] = React.useState<ISourceForm>((state as any)?.source ?? defaultSource);
```

Comment:
Changed to new interface because react can't handle changing state from `undefined` to a value...